loops.js

```diff
@@ -33,7 +42,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
-
+for (let i = 0; i < 16; i++) {
+    if (i % 2 === 0) {
+        console.log([i] + " is even");
+    } else {
+        console.log([i] + " is oneven");
+    }
+}
```

Comment:
Het is helemaal goed! Alleen misschien interessant om te weten dat de brackets ---> [ ]  bij de variabele i niet nodig zijn het werkt prima in Javascript, maar als je dit doet bij een andere taal zoals Java dan krijg je een error.